File: tests_bench/iterative_fib.py
import time

iterations = 1_000_000
max_fib = 30

start = time.perf_counter()

for i in range(iterations):
    a = 0
    b = 1
    for j in range(max_fib):
        temp = a + b
        a = b
        b = temp

end = time.perf_counter()
print((end - start) * 1000)  

File: tests_bench/iterative_fib.lua

local start = os.clock()
local iterations = 10
local max_fib = 10


for i = 1, iterations do
    local a, b = 0, 1
    for j = 1, max_fib do
        local temp = a + b
        a = b
        b = temp
    end
end

local elapsed = os.clock() - start
print(elapsed * 1000)

